$MirOS: ports/multimedia/mplayer/patches/patch-etc_example_conf,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
--- etc/example.conf.orig	Thu Mar  3 03:11:22 2005
+++ etc/example.conf	Thu May 19 20:34:05 2005
@@ -1,3 +1,4 @@
+# $MirOS: ports/multimedia/mplayer/patches/patch-etc_example_conf,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
 ##
 ## MPlayer configuration file
 ##
@@ -132,7 +133,7 @@
 ## Other (preferred to be default from configfile) switches
 ##
 # Drop frames to preserve audio/video sync.
-framedrop 	= yes
+# framedrop 	= yes
 
 ##
 ## cache settings
